src/test/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinterTest.java

```diff
@@ -66,8 +71,31 @@ public void testParseFileWithError() throws Exception {
             final String expected = "[ERROR:0] Javadoc comment at column 1 has parse error. "
                     + "Missed HTML close tag 'qwe'. Sometimes it means that close tag missed "
                     + "for one of previous tags.";
-            Assert.assertEquals(expected, ex.getMessage());
+            assertEquals(expected, ex.getMessage());
         }
     }
 
+    @Test
+    public void testCreationOfFakeCommentBlock() throws Exception {
+        final Method createFakeBlockComment =
+                Whitebox.getMethod(DetailNodeTreeStringPrinter.class,
+                        "createFakeBlockComment", String.class);
```

Comment:
We can't call this method through the normal process instead of invoking it via reflection?